model/Operations/Properties/reviews.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# reviews
+
+## Summary
+
+Link to the review reports
+
+## Description
+
+This field provides the links to the review reports
+
+## Metadata
+
+- name: relationType
```

Comment:
```suggestion
- name: reviews
```

This much matched with the property name "reviews" (the name of the top heading and the filename).

(Note that we may need to call it all "review", instead of "reviews", because of a decision in https://github.com/spdx/spdx-3-model/issues/226